gui/default/syncthing/device/shareDeviceIDDialogView.html

```diff
@@ -0,0 +1,39 @@
+<modal id="share-device-id-dialog" status="warning" icon="{{shareDeviceIdParams.icon}}" heading="{{shareDeviceIdParams.heading}}" large="{{ shareDeviceIdParams.method == 'email' ? 'yes' : 'no' }}" closeable="yes">
+  <div class="modal-body" ng-switch="shareDeviceIdParams.method">
+    <p>
+      <span translate>The following text will automatically be inserted into a new message.</span>
+      <span>
+        <span ng-switch-when="email">
+          <span translate>Your email app should open to let you choose the recipient and send it from your own address.</span>
+        </span>
+        <span ng-switch-when="sms">
+          <span translate>Your SMS app should open to let you choose the recipient and send it from your own number.</span>
+        </span>
+      </span>
+      <span translate>You can also copy and paste the content into the message manually.</span>
+    </p>
+    <hr>
```

Comment:
Move down one line, so it only applies for email.

Reply:
Done.